### ISSUE TITLE
Object reference not set to an instance of an object.

### ISSUE THREAD

**Issue by petarpetrovt** [swharden/ScottPlot#189]
Example
```charp
this.ScottPlotUC1 = new ScottPlot.FormsPlot();
```
NuGet: `ScottPlot.WinForms` - `4.0.6`
StackTrace:
```
  at ScottPlot.FormsPlot.PbPlot_SizeChanged(Object sender, EventArgs e)
   at System.Windows.Forms.Control.OnSizeChanged(EventArgs e)
   at System.Windows.Forms.Control.UpdateBounds(Int32 x, Int32 y, Int32 width, Int32 height, Int32 clientWidth, Int32 clientHeight)
   at System.Windows.Forms.Control.UpdateBounds(Int32 x, Int32 y, Int32 width, Int32 height)
   at System.Windows.Forms.Control.SetBoundsCore(Int32 x, Int32 y, Int32 width, Int32 height, BoundsSpecified specified)
   at System.Windows.Forms.Control.ScaleControl(SizeF factor, BoundsSpecified specified)
   at System.Windows.Forms.Control.ScaleControl(SizeF includedFactor, SizeF excludedFactor, Control requestingControl)
   at System.Windows.Forms.Control.Scale(SizeF includedFactor, SizeF excludedFactor, Control requestingControl)
   at System.Windows.Forms.Control.ScaleChildControls(SizeF includedFactor, SizeF excludedFactor, Control requestingControl, Boolean updateWindowFontIfNeeded)
   at System.Windows.Forms.ContainerControl.Scale(SizeF includedFactor, SizeF excludedFactor, Control requestingControl)
   at System.Windows.Forms.ContainerControl.PerformAutoScale(Boolean includedBounds, Boolean excludedBounds)
   at System.Windows.Forms.ContainerControl.PerformNeededAutoScaleOnLayout()
   at System.Windows.Forms.ContainerControl.OnLayoutResuming(Boolean performLayout)
   at System.Windows.Forms.Control.ResumeLayout(Boolean performLayout)
   at ScottPlot.FormsPlot.InitializeComponent()
   at ScottPlot.FormsPlot..ctor()
```

I would suggest the adoption of c# [8.0 null reference types](https://devblogs.microsoft.com/dotnet/embracing-nullable-reference-types/).
As this is not the first time i report this problem :)

**Comment by swharden** [comment]
Hi @petarpetrovt thanks for reporting this. Unfortunately I cannot reproduce this error - the control installs fine for me.

Are you using this in a .NET Framework application? What version of the framework?

**Comment by swharden** [comment]
I found your previous issue (#138) which is the identical problem. I realized you're using WinForms with .NET Core 3.0, and by creating a .NET Core 3.0 WinForms application I was able to reproduce this error. This error only occurs in .NET Core 3.0 WinForms projects.

For simplicity I'll discuss the solution in #138. Thanks again for mentioning this!

**Comment by swharden** [comment]
I believe this is fixed in 4.0.7 (now live on NuGet)
